CONTRIBUTING.md

```diff
@@ -78,9 +78,9 @@ We have a list of [beginner friendly issues](https://github.com/facebook/docusau
 
 ### Proposing a change
 
-If you would like to request a new feature or enhancement but are not yet thinking about opening a pull request, you can also file issues as [feature requests or enhancements](https://github.com/facebook/docusaurus/labels/enhancement)..
+If you would like to request a new feature or enhancement but are not yet thinking about opening a pull request, you can also file an issue with [feature template](https://raw.githubusercontent.com/facebook/docusaurus/master/.github/ISSUE_TEMPLATE/feature.md).
 
-If you intend to change the public API (e.g., something in `siteConfig.js`), or make any non-trivial changes to the implementation, we recommend [filing an issue](https://github.com/facebook/docusaurus/issues/new?title=%5BProposal%5D) that includes `[Proposal]` in the title. This lets us reach an agreement on your proposal before you put significant effort into it. These types of issues should be rare.
+If you intend to change the public API (e.g., something in `siteConfig.js`), or make any non-trivial changes to the implementation, we recommend filing an issue with [proposal template](https://raw.githubusercontent.com/facebook/docusaurus/master/.github/ISSUE_TEMPLATE/proposal.md) and includes `[Proposal]` in the title. This lets us reach an agreement on your proposal before you put significant effort into it. These types of issues should be rare.
```

Comment:
Use this link instead for proposal template: https://github.com/facebook/docsuaurus/issues/new?template=proposal.md